cmd/crictl/container.go

```diff
@@ -302,16 +332,24 @@ func ContainerStatus(client pb.RuntimeServiceClient, ID string) error {
 		if r.Status.Metadata.Name != "" {
```

Comment:
What do you mean by other fields? there are only name and attempt in ContainerMetadata.